de.metas.acct.base/src/main/java/de/metas/acct/process/GL_Journal_GenerateYearEnding.java

```diff
@@ -201,6 +201,7 @@ private void createFacts_ProfitAndLoss_BroughtForward(final GL_Journal_Builder g
 
 		final IQueryAggregateBuilder<I_Fact_Acct, I_C_ElementValue> aggregateOnAccountBuilder = queryBL.createQueryBuilder(I_Fact_Acct.class, this)
 				.addEqualsFilter(I_Fact_Acct.COLUMN_AD_Client_ID, getAD_Client_ID())
+				.addEqualsFilter(I_Fact_Acct.COLUMN_AD_Org_ID, p_C_AcctSchema.getAD_Org_ID())
```

Comment:
@metas-rc talked with @metas-mk about this.... you must filter by GL journal's AD_Org_ID and not C_AcctSchema's AD_Org_ID because that one is always ZERO.